test/run_test.py

```diff
@@ -677,6 +704,60 @@ def run_doctests(test_module, test_directory, options):
     return result
 
 
+def print_log_file(test, file_path):
+    with open(file_path, "r") as f:
+        print_to_stderr("")
+        print_to_stderr(f'PRINT LOG FILE of {test} ({file_path})')
```

Comment:
Nit: I was under the wrong impression that f-string only works with double quote, but it works with single quote too. You might want to fix the quote here though because I think the linter will complain